src/pc/gfx/gfx_dxgi.cpp

```diff
@@ -685,6 +685,11 @@ HWND gfx_dxgi_get_h_wnd(void) {
 void gfx_dxgi_shutdown(void) {
 }
 
+// TODO: is this even right????
```

Comment:
Please test this

Reply:
:+1: Will test it now.

Reply:
Tested on Windows 10, targeting D3D12. Works as intended, will be removing the comment.